src/escpos/printer/win32raw.py

```diff
@@ -70,36 +77,75 @@ def is_usable() -> bool:
         return is_usable()
 
     @dependency_win32print
-    def __init__(self, printer_name=None, *args, **kwargs):
+    def __init__(self, printer_name: str = "", *args, **kwargs):
         """Initialize default printer."""
         Escpos.__init__(self, *args, **kwargs)
-        if printer_name is not None:
-            self.printer_name = printer_name
-        else:
-            self.printer_name = win32print.GetDefaultPrinter()
-        self.hPrinter = None
-        self.open()
+        self.printer_name = printer_name
+        self.job_name = ""
+
+    @property
```

Comment:
```suggestion
    @property
    @dependency_win32print
```
Since this property is also using win32, we should guard it.